docs/testing.rst

```diff
@@ -0,0 +1,60 @@
+=======
+Testing best practices
+=======
+
+Using setUpClass
+================
+
+Doing a lot of work in the ``setUp`` call of a test class means that it will be run on every test. This
+quickly adds a lot of run time to the tests. Some things that can be easily moved to ``setUpClass`` are domain
+creation, user creation, or any other static models needed for the test.
+
+Using SimpleTestCase
+====================
+
+The SimpleTestCase runs tests without a database. Many times this can be achieved through the use of the `mock
+library <http://www.voidspace.org.uk/python/mock/>`_. A good rule of thumb is to have 80% of your tests be unit
+tests that utilize ``SimpleTestCase``, and then 20% of your tests be integration tests that utilize the
+database and ``TestCase``.
+
+
+Tearing down tests
+==================
+
+It is good practice to ensure that all objects you have created in the test database are deleted when the test
+class finishes running. This often happens in the ``tearDown`` method or the ``tearDownClass`` method.
+However, unneccessary cleanup "just to be safe" can add a large amount of time onto your tests.
+
+Combining test classes
+======================
+
+Sometimes classes share the same base class and inherit the ``setUpClass`` function. Below is an example:
+
+.. code:: python
+
+    class MyBaseTestClass(TestCase):
```

Comment:
This might be a bad example. I think that you would want the base class to inherit from object and have the child classes inherit tow classes http://stackoverflow.com/questions/4566910/abstract-test-case-using-python-unittest